This fixes weird check_headers failures when it also matches things like 'const' which are obviously allowed in the public headers and cause check failures due to these false positives.
---
 src/check_headers |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Index: vlc-clean/src/check_headers
===================================================================
--- vlc-clean.orig/src/check_headers
+++ vlc-clean/src/check_headers
@@ -6,7 +6,7 @@ cd "$(dirname "$0")" || exit $?
 # Look for configure #defines in public headers.
 # There are incorrect, as external users don't have our <config.h>.
 regexp="$(cat ../config.h.in | \
-	sed -n -e 's/^#undef \([A-Z0-9_]*\)$/\1/p' | \
+	sed -n -e 's/^#undef \([[:upper:][:digit:]_]*\)$/\1/p' | \
 	grep -v 'WORDS_BIGENDIAN' | \
 	xargs | \
 	sed -e 's/ /\\\|/g')"
